vNext/global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "2.0.0-preview3-006670"
```

Comment:
Also, please add to the readme how to download the version of the correct tooling to open in VS